web-jehlomat/src/screens/EditaceUzivatele/EditaceUzivatele.tsx

```diff
@@ -18,15 +18,17 @@ import { media } from 'utils/media';
 import { userState } from 'store/user';
 import { AccessDenied } from 'screens/Organizace/403';
 import { isStatusGeneralSuccess } from 'utils/payload-status';
-import { primaryDark } from 'utils/colors';
+import { primary, primaryDark } from 'utils/colors';
 import TextInput from 'Components/Inputs/TextInput';
 import PrimaryButton from 'Components/Buttons/PrimaryButton/PrimaryButton';
 import SelectInput from 'Components/Inputs/SelectInput/SelectInput';
+import TextButton from 'Components/Buttons/TextButton/TextButton';
+import { useConfirmationModalContext } from 'context/confirmation-modal-context';
 
 const validationSchema = yup.object({
     email: yup.string().email('Email nemá správný formát.').required('Email je povinné pole'),
     username: yup.string().required('Uživatelské jméno je povinné pole'),
-    teamId: yup.number().required('Tým je povinné pole'),
+    teamId: yup.number().required('Tým je povinné pole').nullable(),
```

Comment:
Does it work as expected? At first glance, it looked strange, so I've googled it and found the following discussion; but it's a bit outdated, last comment 10 month ago, could you please confirm that it works as expected?
https://github.com/jquense/yup/issues/374

Reply:
yes, it is works as expected.. without the `nullable` it would in case of empty value show different error that we want (it would complaint about the value not being a number).. this way it shows user correct error message